src/test/java/com/icthh/xm/tmf/ms/activation/web/rest/v4/ServiceResourceApiImplTest.java

```diff
@@ -39,7 +40,7 @@
 @RunWith(SpringRunner.class)
 @WebMvcTest(controllers = ServiceResourceApiController.class, secure = false)
 @ContextConfiguration(classes = {ServiceResourceApiController.class, ServiceResourceApiImpl.class,
-    ExceptionTranslator.class, ServiceMapperImpl.class})
+    ExceptionTranslator.class, ServiceResourceHelperServiceImpl.class, ServiceMapperImpl.class})
 public class ServiceResourceApiImplTest {
```

Comment:
Can we extend tests with new functionality?

Reply:
I've got 100% coverage for all the changes: 
<img width="1806" alt="image" src="https://user-images.githubusercontent.com/895187/118639702-dc35c200-b7e0-11eb-8fcd-6a61ea835436.png">